python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1725,12 +1739,17 @@ def run(
                     "It's not possible to stream and detach a container at "
                     "the same time."
                 )
+
+        if preserve_fds:
+            pass_fds = range(3, 3 + preserve_fds)
```

Comment:
I wouldn't mind an explanation in a code comment for this line. That should speed up the comprehension for future readers